src/sentry/ingest/inbound_filters.py

```diff
@@ -200,8 +200,7 @@ def _get_filter_settings(project_config, flt):
 )
 
 
-class _LegacyBrowserFilterSerializer(serializers.Serializer):
-    active = serializers.BooleanField()
+class _LegacyBrowserFilterSerializer(_FilterSerializer):
```

Comment:
Quick fix for better class hierarchy as `LegacyBrowserFilter` is subclass of generic filter